sdk/core/azure-core/CHANGELOG.md

```diff
@@ -3,14 +3,19 @@
 ## 1.8.0-beta.4 (Unreleased)
 
 ### Features Added
+- Aligned OpenTelemetry attributes to align with OpenTelemetry 1.17.0.
```

Comment:
Core does not know about OpenTelemetry, is it still a good wording/does it still apply?

Reply:
There are changes to Azure Core to align core with OpenTelemetry 1.17.0. 

IMHO, These changes should be called out in the Azure Core changelog, since they're in Azure Core.